.ci/ci-fast-return.sh

```diff
@@ -374,6 +480,124 @@ doc.md" "$matches"
 
 }
 
+testIsLabelSet() {
```

Comment:
Non-blocking question: Where are those tests picked up?

Reply:
They are for updating the scirpt: they aren't run by the CI (since if they fail, the CI is dead ;)